---
title: MSSQLSERVER_8680 | Microsoft Docs
ms.custom: ''
ms.date: 04/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: supportability
ms.topic: language-reference
helpviewer_keywords:
- 8680 (Database Engine error)
ms.assetid: 79f7270e-e4df-4f4e-bc78-2dcad6a9f140
author: MashaMSFT
ms.author: mathoma
ms.openlocfilehash: a6f7d15cb13546e19fd234d4b738d11d085b1360
ms.sourcegitcommit: 58158eda0aa0d7f87f9d958ae349a14c0ba8a209
ms.translationtype: HT
ms.contentlocale: es-ES
ms.lasthandoff: 03/30/2020
ms.locfileid: "68120574"
---
# <a name="mssqlserver_8680"></a>MSSQLSERVER_8680
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
  
## <a name="details"></a>Detalles  
  
|||  
|-|-|  
|Nombre de producto|SQL Server|  
|Id. de evento|8680|  
|Origen de eventos|MSSQLSERVER|  
|Componente|SQLEngine|  
|Nombre simbólico|REMOTE_ERR|  
|Texto del mensaje|Error interno del procesador de consultas: error inesperado al procesar una fase de consulta remota.|  
  
## <a name="explanation"></a>Explicación  
Se ha producido un error interno al procesar la consulta.  
  
## <a name="user-action"></a>Acción del usuario  
Póngase en contacto con el servicio de soporte técnico y atención al cliente de Microsoft (CSS). Tenga a mano la descripción del error, los seguimientos y los volcados (si dispone de ellos).  
  
